Assets/Level/Scripts/Player/PlayerMover.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerMover : MonoBehaviour
+{
+    [SerializeField] private float _speed;
+
+    public float Speed { get => _speed; set => _speed = value; }
+
+    private void Update()
+    {
+        Vector3 movement = new Vector3(Input.GetAxis("Horizontal"), Input.GetAxis("Vertical"), 0);
```

Comment:
https://trello.com/c/UAeMGORw/17-%D0%BE%D1%82%D0%BA%D0%B0%D0%B7-%D0%BE%D1%82-%D1%81%D1%82%D1%80%D0%BE%D0%BA%D0%BE%D0%B2%D1%8B%D1%85-%D0%BB%D0%B8%D1%82%D0%B5%D1%80%D0%B5%D0%BB%D0%BE%D0%B2